web/scripts/stripe/stripe.js

```diff
@@ -0,0 +1,61 @@
+/*
+sk key : sk_test_tjoHHaJQvf1Y4CoAEpWBWr0w
+publish key :pk_test_4VicAuG5Ou9zuqA4LVIRI0dC
+ from stripe.com
+
+*/
+// Refered to the video https://www.youtube.com/watch?v=FhBtM70QZKM
+// https://stripe.com/docs/stripe-js/elements/quickstart
+// npm install stripe
+// installing body parser
+// $ npm install body-parser
+// API
+
+// npm install express
+
+// install hbs
+// npm install hbs
```

Comment:
Please delete the comments from line 1-17. We can keep track of the keys you have comments in our google drive instead of having them as comments in the code.